.circleci/config.yml

```diff
@@ -7,8 +7,172 @@ orbs:
 
 
 jobs: 
+   build-and-push-docker-image:
+    environment:
+      - IMAGE: taraxa-node
+    machine:
+      image: ubuntu-1604:202007-01
```

Comment:
We should be using a newer image
https://circleci.com/docs/2.0/configuration-reference/#available-machine-images